pkg/eventprocessor/impl/webhook.go

```diff
@@ -76,14 +80,163 @@ func sendWebhookRequest(ctx context.Context, url string, body interface{}) error
 	return nil
 }
 
+type chain struct {
+	ID               int64
+	Name             string
+	ContractAddr     common.Address
+	TBLDocsURL       string
+	BlockExplorerURL string
+	SupportsNFTView  bool
```

Comment:
Some block explorers like Etherscan have started to provide an NFT view. I believe this feature will be added to others eventually. Using this flag we can provide a link to the NFT view if available.